test/uniport-tests.js

```diff
@@ -0,0 +1,502 @@
+/* -------------------------------- REQUIRES -------------------------------- */
+
+var async       = require("async");
+var assert      = require('assert');
+var assert      = require('assert');
+var UInt160     = require("ripple-lib").UInt160;
+var Remote      = require('ripple-lib').Remote;
+var Server      = require('ripple-lib').Server;
+var Request     = require('ripple-lib').Request;
+var testutils   = require("./testutils");
+var config      = testutils.init_config();
+var http        = require('http');
+var request     = require('request');
+
+/* --------------------------------- CONFIG --------------------------------- */
+
+// So we can connect to websocket ports
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+
+var configurations = {
+  // All the tests run on one invokation of rippled. It's possible for you to:
```

Comment:
You might want to hoist that comment out of the configuration variable.

Also, it's often easier for everyone if long comments use `/*` and `*/`